src/test_memcachedsolution.cpp

```diff
@@ -155,11 +161,28 @@ class BaseMemcachedSolutionTest : public ::testing::Test
   {
     for (int ii = 0; ii < astaire_instances; ++ii)
     {
-      _astaire_instances.emplace_back(new AstaireInstance(ASTAIRE_PORT));
+      std::string ip = "127.0.0." + std::to_string(ii + 1);
+      _astaire_instances.emplace_back(new AstaireInstance(ip, ASTAIRE_PORT));
       _astaire_instances.back()->start_instance();
     }
   }
 
+  /// Creates and starts up a dnsmasq instance to allow the store to find
+  /// Astaire instances.
+  static void create_and_start_dns_for_astaire(int astaire_instances)
```

Comment:
Would it be nicer if this took a `vector` of `AstaireInstance`s?  That way you don't need to duplicate the IP address calculation.